inputs_replace.py

```diff
@@ -0,0 +1,30 @@
+from mwrogue.esports_client import EsportsClient
+from mwrogue.auth_credentials import AuthCredentials
+from mwcleric.errors import RetriedLoginAndStillFailed
+import mwparserfromhell
+
+credentials = AuthCredentials(user_file="bot")
+site = EsportsClient("lol", credentials=credentials, max_retries=0)
+
+response = site.cargo_client.query(
+    tables="Teamnames=T, Teamnames__Inputs=I, Teamnames__Inputs=I2",
+    where="I._value LIKE '%,%' AND I2._value NOT LIKE '%,%'",
+    join_on="T._ID=I._rowID, I._rowID=I2._rowID",
+    fields="I._value=BadInput, I2._value=GoodInput, T.Link",
```

Comment:
you can also grab T._pageName=Page here, and then remove the bad key in your main body code - I was gonna go back later and do that cos I didn't want to do more text parsing on the wiki but since you put the actual query here now, may as well